buildtest/settings/config.yml

```diff
@@ -31,6 +31,7 @@ system:
           shell: zsh
     # compiler block
     compilers:
+      modulepath: ['/tmp', '/var/']
```

Comment:
please remove this from the default configuration. It is good for testing but should not be in the configuration. 

What you can do is put these two lines in comment and provide a brief description such as